Packs/EWS/Integrations/EWSO365/EWSO365.py

```diff
@@ -2066,9 +2071,23 @@ def parse_incident_from_item(item):
                     attached_email_bytes = attached_email.as_bytes()
                     chardet_detection = chardet.detect(attached_email_bytes)
                     encoding = chardet_detection.get('encoding', 'utf-8') or 'utf-8'
+                    try:
+                        # Trying to decode using the detected encoding
+                        data = attached_email_bytes.decode(encoding)
+                    except UnicodeDecodeError:
+                        # In case the detected encoding fails apply the default encoding
+                        demisto.debug(f'Could not decode data using detected encoding:{encoding}, retrying using '
+                                      f'default encoding: {default_encoding}')
+
+                        try:
+                            data = attached_email_bytes.decode(default_encoding)
+                        except Exception as e:
+                            demisto.debug(f'attached_email bytes:\n{attached_email}')
+                            demisto.debug(e)
+
                     file_result = fileResult(
                         get_attachment_name(attachment.name) + ".eml",
-                        attached_email_bytes.decode(encoding),
+                        data,
```

Comment:
you might get a `used before assigned` error in case both encoding options fails